libs/state/docs/api/operators/select-slice.md

```diff
@@ -0,0 +1,39 @@
+# selectSlice
+
+Returns an Observable that emits only the provided `keys` emitted by the source Observable. Each key will get
+filtered to only emit _defined_ values as well as checked for distinct emissions.
+
+_Example_
+
+```TypeScript
+import { of, Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
+import { selectSlice } from 'rx-angular/state';
+
+interface MyState {
+   title: string;
+   items: string[];
+   panelOpen: boolean;
+}
+const state$: Observable<MyState> = of({
+  title: 'myTitle',
+  items: ['foo', 'bar'],
+  panelOpen: true
+});
+const slice$ = state$.pipe(selectSlice('items', 'panelOpen')).pipe(tap(console.log)).subscribe();
```

Comment:
Just a thought:
It might be a bit confusing for some users.
- state.select('bar', 'foo'); will return nested property
- selectSlice('items', 'pannelOpen'); will return properties from the same level.
In addition distinctUntilSomeChnaged accepts strict array (without rest) of keys and is also mostly for managing viewModel.

Maybe just an array?